package.json

```diff
@@ -99,4 +99,4 @@
   "version": "2.0.0",
   "repository": "https://github.com/sapcc/elektra.git",
   "main": "index.js"
-}
+}
```

Comment:
why there is a change on this file? Empty line?

Reply:
Change is reverted already. https://github.com/sirajudheenam/elektra/blob/ec2_credential_fix_26_oct_23/package.json#L102